src/Symfony/Component/Validator/Tests/Fixtures/ConstraintWithRequiredArgument.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Fixtures;
+
+use Symfony\Component\Validator\Attribute\NamedArgumentConstraintConstructor;
+use Symfony\Component\Validator\Constraint;
+
+#[\Attribute]
+#[NamedArgumentConstraintConstructor]
```

Comment:
what about adding it to the constructor instead of the class?
then the name could be `HasNamedArguments`